### DISCUSSION TITLE
--fusion_mode with openMVS_sample

### DISCUSSION THREAD

**Discussion by AdrianKriger** [cdcseacave/openMVS#699] [Q&A] [answered]
Hi;

I am attempting to follow the [Scalable pipeline](https://github.com/cdcseacave/openMVS/issues/438) through extending @FlachyJoe `s [MvgMvsPipeline.py](https://github.com/cdcseacave/openMVS/blob/master/MvgMvsPipeline.py) with
```python
["Densify point cloud",      # 14
os.path.join(OPENMVS_BIN, "DensifyPointCloud"),
["-i", "sceneGeo.mvs", "--resolution-level", "1", "--fusion-mode", "1",  "--number-views", "5", "-w", "%mvs_dir%"]],
```
which will be followed by:
```python
["Split scene",          # 15
os.path.join(OPENMVS_BIN, "DensifyPointCloud"),
["-i", "sceneGeo.mvs", "--sub-scene-area", "660000", "--max-threads", "4", "-w", "%mvs_dir%"]],
["Export scene",          # 16
os.path.join(OPENMVS_BIN, "DensifyPointCloud"),
[ {another function will loop through each `sceneGeo_xxxx.mvs` here} "--dense-config-file", "Densify.ini", "--resolution-level", "1", "--number-views-fuse", "2", "--max-threads", "4", "-w", "%mvs_dir%"]],
```

Unfortunately I don't get that far. The cmd returns:
```(base) PS C:\openMVG_MVS> python MvgMvsPipeline_Pix4D100.py C:\Adrian\openMVG_MVS\_100\images_100 C:\openMVG_MVS\_100\result
# Using input dir:  C:\openMVG_MVS\_100\images_100
#   Output dir:  C:\openMVG_MVS\_100\result
# Steps:  [14, 15, 16]
#14. Densify point cloud
Cmd: C:\Adrian\openMVG_MVS\openMVS\bin\DensifyPointCloud -i sceneGeo.mvs --resolution-level 1 --fusion-mode 1 --number-views 5 -w C:\Adrian\openMVG_MVS\_100\result\mvs
18:16:48 [App     ] unrecognised option '--fusion-mode'
# Pipeline end #
```
I am using the binaries at [openMVS_sample](https://github.com/cdcseacave/openMVS_sample). 
I cannot build my own. the computer literally says while executing:
```
#install dependencies through vcpkg
C:\dev\vcpkg install zlib boost-iostreams boost-program-options boost-system boost-serialization eigen3 cgal[core] opencv glew glfw3  --triplet=x64-windows
```
- Blue Screen: "Your PC has en-counted a problem and needs to restart".

Is `fusion-mode` not an option with the pre-built [binaries](https://github.com/cdcseacave/openMVS_sample/tree/master/bin)?
Is there a specific [openMVG](https://github.com/openMVG/openMVG) parameter that `--fusion-mode` needs?

**Comment by cdcseacave** [comment, marked as answer]
sample repo is up to date with master, and this feature is not in master, use the binary in the autobuild repo

**Reply by AdrianKriger** [reply]
I am sorry for not replying sooner. Forgive my tardiness. I cannot continue with this now. I will accept the answer and when better able will edit the post with a proper statement (success, etc.). 

Thank you for the reply. ~~ the dense reconstruction is spectacular!
